pkgs/applications/misc/organicmaps/default.nix

```diff
@@ -0,0 +1,89 @@
+{ lib
+, mkDerivation
+, fetchFromGitHub
+, fetchpatch
+, cmake
+, ninja
+, pkg-config
+, which
+, python3
+, makeWrapper
+, qtbase
+, qtsvg
+, libGLU
+, zlib
+}:
+
+mkDerivation rec {
+  pname = "organicmaps";
+  version = "2021-05-15";
+
+  src = fetchFromGitHub {
+    owner = "organicmaps";
+    repo = "organicmaps";
```

Comment:
```suggestion
    repo = pname;
```

Reply:
Is there a consensus on whether to use the pname in the src? I'm personally for not using it, since while the pname and src are certainly related, the src does not necessarly depend on the pname, as opposed to the version.

Reply:
Using pname in like this is a bit discouraged because the repo name can vary.